test/cypress/e2e/example11.cy.ts

```diff
@@ -22,6 +22,20 @@ describe('Example 11 - Batch Editing', () => {
     cy.get('h3').should('contain', 'Example 11 - Batch Editing');
   });
 
+  it('should input values directly in input of datepicker', () => {
+    cy.get(`[style="top: ${GRID_ROW_HEIGHT * 0}px;"] > .slick-cell:nth(5)`)
+      .click();
+
+    cy.get(`.input-group-editor`)
+      .focus()
+      .type('{backspace}'.repeat(10))
```

Comment:
might be easier to simply use Ctrl+Backspace

Reply:
{ctrl}{backspace} didn't work for me. But I doubt there is any con with this approach. So if you're fine I'd just keep it that way

Reply:
ok sure whatever works is fine, just thought the alternative was shorter but if it doesn't work then too bad

Reply:
the E2E test still fails though, should we use `{enter}` key after typing the date?